test/dummy/app/models/orange.rb

```diff
@@ -0,0 +1,9 @@
+class Orange < ActiveRecord::Base
+  has_paper_trail class_name: "OrangeVersion"
+  default_scope -> { where(name: "orange") }
+
+  self.primary_key = :uuid
+  before_create do
+    self.uuid ||= SecureRandom.uuid
+  end
+end
```

Comment:
Instead of calling this model `Orange`, what about `CustomPrimaryKeyRecord`?

We have so many models in our test suite it's getting hard to remember how they differ.